weasyl/define.py

```diff
@@ -576,6 +576,20 @@ def convert_date(target=None):
     return result[1:] if result and result[0] == "0" else result
 
 
+def convert_iso_date(target=None):
+    """
+    Converts a unix timestamp to an ISO 8601 date (yyyy-mm-dd). If no target is passed,
+    the current date is returned.
+
+    Note that the date is converted to localtime (via ``convert_to_localtime``).
```

Comment:
It shouldn’t be, right? The original implementation was UTC, and the display/save process shouldn’t depend on the admin’s timezone. (We shouldn’t be storing it as a timestamp at all, but that’s a separate mistake. :)

Reply:
I would tend to agree here, though I suppose my reasoning behind this decision was that I was following the same convention as the surrounding functions seeming to use ``convert_to_localtime()``.

Should it instead be something akin to ``date = arrow.get(target) if target else arrow.utcnow()`` and take ``target`` to be a UTC/unix timestamp, instead of just ``date = convert_to_localtime(target)``?

Reply:
`iso8601` uses `datetime.datetime.utcfromtimestamp(unixtime - _UNIXTIME_OFFSET)`. Maybe this could go next to that and be called `iso8601_date`?